src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -26,12 +51,102 @@ public void onDisable(){
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
         if (label.equalsIgnoreCase("simple")){
-            if (sender instanceof Player){
-                Player me = (Player)sender;
+            if (sender instanceof Player) {
+                Player me = (Player) sender;
                 me.sendMessage("Plugin Working");
                 return true;
             }
-        }
+                if (label.equalsIgnoreCase("hawkeye")){//check command label
+                    if(args[0].equalsIgnoreCase("tnt")){//check if the second word is tnt
+                        if(sender instanceof Player){
+                            Player me =(Player)sender;
+                            PlayerInventory inventory = me.getInventory();
+                            for(int i = 0; i<inventory.getSize(); i++){
+                                ItemStack item = inventory.getItem(i);
+                                        if(item == null){
+                                            continue;
+                                        }
+                            if(isArrow(item)==true){
+                                item.setType(Material.TIPPED_ARROW);
+                                item.setAmount(item.getAmount());
+                                PotionMeta potionMeta = (PotionMeta)item.getItemMeta();
+                                potionMeta.setBasePotionData(new PotionData(PotionType.JUMP));
+                                int duration =10;
```

Comment:
as these values are repeating why not make them class level constants aka private static final

Reply:
they are different in the different uses